extreme_classification/extreme_ml.ipynb

```diff
@@ -33,7 +33,7 @@
   },
```

Comment:
<p>Can you change the env to  'us-west1-gcp' here as well? thanks !</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>